media-video/motion/motion-4.4.0.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2021 Gentoo Authors

```

Comment:
Bump to match the current year. We have tools that does this automatically, e.g. repoman or pkgdev. 
```suggestion
# Copyright 1999-2022 Gentoo Authors

```